input/number/reports.md

```diff
@@ -0,0 +1,28 @@
+## This page should be used for casual reporting of issues.
+
+For help on formatting, see: 
```

Comment:
Remove this section regarding formatting

Reply:
Will make (and resolve) changes once/if we agree my idea has some value.